modules/site/src/main/resources/microsite/data/courses/DrumoreLinks.yaml

```diff
@@ -0,0 +1,66 @@
+courseName: Drumore Links
+skinned: false
+holes:
+  - id: e9d88d48-361e-41d2-bd48-0bf29109c130
+    course: Drumore Links
+    number: 1
+    par: 4
+    replayName: "1"
```

Comment:
Do we have the real replay names available for these holes? Not a blocker but would be nice.

Reply:
@cranst0n I do not, unfortunately no one paid attention during the tournament and I'm not sure how I get them now.

Reply:
You can find them in the in-game Replay section after you've played the hole. It's a shame PD doesn't make this info easier to find/share but their track record for making meaningful improvements is what it is....poor. But as I said, this isn't a blocker and can be added at a later date.